src/Enum.php

```diff
@@ -87,7 +87,7 @@ public function __toString()
      *
      * @return bool
      */
-    final public function equals($variable = null)
+    final public function equals($variable = null): bool
```

Comment:
What about adding other return types?

Reply:
Please read the discussion above (basically it would be a BC break)

Reply:
Ah, okay. Still working on my first cup of ☕️